core/platform/datastore/gae_datastore_services.py

```diff
@@ -31,39 +31,51 @@
 from google.appengine.datastore import datastore_stub_util
 from google.appengine.ext import ndb
 
+from typing import ( # isort:skip # pylint: disable=unused-import
+    Any, Callable, Iterator, List, Optional, Text, Tuple)
+
+MYPY = False
+if MYPY:
+    from mypy_imports import * # pragma: no cover # pylint: disable=import-only-modules,wildcard-import,unused-wildcard-import
+
 transaction_services = models.Registry.import_transaction_services()
-Model = ndb.Model
-Key = ndb.Key
-Property = ndb.Property
-Query = ndb.Query
 
-BooleanProperty = ndb.BooleanProperty
-DateProperty = ndb.DateProperty
-ComputedProperty = ndb.ComputedProperty
-DateTimeProperty = ndb.DateTimeProperty
-FloatProperty = ndb.FloatProperty
-IntegerProperty = ndb.IntegerProperty
-JsonProperty = ndb.JsonProperty
-UserProperty = ndb.UserProperty
+
+Model = ndb.Model # type: Any
+Key = ndb.Key # type: Any
```

Comment:
@brianrodri ndb isn't type annotated and we don't have type-stubs for it - that's why this was @hardikkat24's primary decision based on options provided by mypy to handle such cases. Though this is also in discussion [here](https://docs.google.com/document/d/1LEWKK2H303eHmWg1hHN2EMIlnewB6EnEyrQoReY-M6s/edit?ts=60f82765#) and @hardikkat24 will provide an update on this in case he finds a better option viable.

Reply:
Changed the approach. Added stubs and used ndb.Key